frontend/javascripts/oxalis/view/right-menu/trees_tab_view.js

```diff
@@ -181,13 +184,54 @@ export async function importTracingFiles(files: Array<File>, createGroupForEachF
       }
     };
 
+    const tryParsingFileAsZip = async file => {
+      try {
+        const zipFile = await JSZip().loadAsync(readFileAsArrayBuffer(file));
+        const nmlFileName = Object.keys(zipFile.files).find(
+          key => _.last(key.split(".")) === "nml",
+        );
+        const nmlFile = await zipFile.file(nmlFileName).async("blob");
+        const nmlImportActions = await tryParsingFileAsNml(nmlFile);
+
+        const dataFileName = Object.keys(zipFile.files).find(
+          key => _.last(key.split(".")) === "zip",
+        );
+        if (dataFileName) {
+          const dataBlob = await zipFile.file(dataFileName).async("blob");
+          const dataFile = new File([dataBlob], dataFileName);
+
+          const { tracing, dataset } = Store.getState();
+          const newLargestSegmentId = await importTracing(tracing, dataFile);
+
+          if (tracing && tracing.volume) {
+            Store.dispatch(importTracingAction()); // TODO put all in one Action, which gets consumed twice (volume tracing reducer + undo save saga)
```

Comment:
Should the two actions be dispatched separately by the `ImportTracingAction`?

Reply:
Let's discuss this in person!